plugins/backend-trace-acceptor/src/Cardano/BM/Backend/TraceAcceptor.lhs

```diff
@@ -185,9 +193,14 @@ clientThread sbtrace h = handleError TraceAcceptorClientThreadError pProc
       bs <- BS.hGetLine h -- payload
       unless (BS.null bs) $ do
         let hname = decodeUtf8 hn
-        case eitherDecodeStrict bs of
-          Right lo ->
-            traceWith sbtrace (loName lo, lo)
+        -- To reduce network traffic, TraceForwarder could send
+        -- list of log items as compressed ByteString, check it.
+        bsToDecode <- getAcceptGZip config >>= \case
+                        Nothing   -> tryToUnzip bs -- By default assume it's compressed.
+                        Just isGZ -> if isGZ then tryToUnzip bs else pure bs
+        case eitherDecodeStrict bsToDecode of
```

Comment:
Sometimes, when I test it using `example-complex` and `example-acceptor`, I see freezing here.

Reply:
The compressing/decompressing mechanism is disabled now. It will be fixed in the future.